modules/openapi-generator/src/main/java/org/openapitools/codegen/templating/mustache/TrimLineBreaksLambda.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
for new files, we don't need to add line 3

Reply:
done